projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -128,6 +129,21 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             " no guarantee that files created during a local data job execution"
             " will  be present or absent for the next execution.",
         )
+        config_builder.add(
+            PYTHON_VERSION,
+            None,
+            False,
+            "The python version, which is to be used for the data job deployment."
+            " This configuration is to be set in the [job] section of the config"
```

Comment:
We should explicitly say, this configuration can only be set in [job] section of the config and will be ignored if set anywhere else (e.g environment variable). 

As this is the current state.

Reply:
Let's update JobConfigIniPlugin to set the value.
This way when inspecting (e.g with vdk debug) people can see what value is configured.

Reply:
Hi, @tozka 

Updated the description and moved the configuration to the JobConfigIniPlugin.

Reply:
I meant that there should be 

```
config_builder.set_value(
                "python_version",
                job_config.get_python_version(),
            )
```

Reply:
Ah, I see. In such case, should the definition remain in JobConfigIniPlugin, or should I return it to the CoreConfigDefinitionPlugin plugin?

Reply:
It does not matter at this stage too much. 

But Job Config seems more appropriate since it's a python version of a job not just the general SDK.

Reply:
I've added the part to set the value.